booking/src/main/java/org/dcsa/conformance/standards/booking/BookingScenarioListBuilder.java

```diff
@@ -78,7 +80,24 @@ private BookingScenarioListBuilder thenAllPathsFrom(BookingState bookingState) {
   private BookingScenarioListBuilder thenAllPathsFrom(
       BookingState bookingState, BookingState originalBookingState) {
     return switch (bookingState) {
-      case CANCELLED, COMPLETED, DECLINED, REJECTED  -> then(shipper_GetBooking(bookingState));
+      case COMPLETED  -> then(
+        shipper_GetBooking(bookingState).thenEither(
+          auc_shipper_sendInvalidBookingAction(SUBMIT_BOOKING_AMENDMENT)
+            .then(shipper_GetBooking(COMPLETED)),
+          auc_shipper_sendInvalidBookingAction(CANCEL_BOOKING)
+            .then(shipper_GetBooking(COMPLETED))
+        )
+      );
+      case DECLINED, CANCELLED, REJECTED -> then(
+        shipper_GetBooking(bookingState).thenEither(
+          auc_shipper_sendInvalidBookingAction(UPDATE_BOOKING)
+            .then(shipper_GetBooking(bookingState)),
+          auc_shipper_sendInvalidBookingAction(SUBMIT_BOOKING_AMENDMENT)
+            .then(shipper_GetBooking(bookingState)),
+          auc_shipper_sendInvalidBookingAction(CANCEL_BOOKING)
+            .then(shipper_GetBooking(bookingState))
+        )
+      );
```

Comment:
Missing a happy path flow (where there is no `auc` step)

Reply:
done..